src/service/controllers/messaging.js

```diff
@@ -35,7 +35,7 @@ exports.handle = (req, res) => {
 exports.getCurrentSession = (req, res) => {
 	var userName = auth.userByCode(req);
 	if (userName) {
-		var session = sessions[userName];
+		var session = sessions[userName] && sessions[userName].length && sessions[userName][0];
```

Comment:
Is there a risk to return always the first session associated to the use in front?